### ISSUE TITLE
Winget AVX2 manifest is out-of-date

### ISSUE THREAD

**Issue by 0x70369** [Alex313031/Thorium-Win#156]
Hello, everyone.

Recently, Thorium M120 was released, so I opened PowerShell and ran the command "winget upgrade -r". However, Thorium wasn't updated. Running the commands "winget show Alex313031.Thorium" and "winget show Alex313031.Thorium.AVX2" generated the following output: 
![image](https://github.com/Alex313031/Thorium-Win/assets/142499149/40e29127-ad84-41ae-8bf8-b6d9553d1978)
![image](https://github.com/Alex313031/Thorium-Win/assets/142499149/f5d39063-be1c-4dd2-a50d-a12b897c17ed)

Please update the manifest, as using a package manager is a lot more convenient and I use winget whenever I'm in Windows and not on Linux.

**Comment by Alex313031** [comment]
@0x70369 I don't manage the winget thing. I think someone else made it. Do you know where it actually downloads from or who maintains it?

**Comment by 0x70369** [comment]
I have no idea on who maintains it, but both options download directly from your GitHub.
Side note: the manifests were updated a few hours after I opened the issue. Maybe whoever maintains them is one of the contributors?

Below are some information about the manifests:
![image](https://github.com/Alex313031/Thorium-Win/assets/142499149/e3f26420-1701-48c7-a5e8-062ba2c35e2c)
![image](https://github.com/Alex313031/Thorium-Win/assets/142499149/950565a1-f36b-48fb-af06-3146f0be8780)
